app/mm-redux/constants/threads.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export default {
+    THREADS_CHUNK_SIZE: 60,
```

Comment:
does it make sense to add this to the other constants where we declare CHUNKS? Also does it make sense to call it `COLLAPSE_REPLY_THREADS_CHUNK_SIZE` or `CRT_CHUNK_SIZE`